http/api/index.php

```diff
@@ -55,6 +56,8 @@
  */
 $app->get( 'public-releases/list', GetAllPublicReleases::class );
 
+$app->get( 'future-visible-releases/list', GetAllFutureVisibleReleases::class);
```

Comment:
Maybe it would be mirror the public-releases as future-releases? I guess that only shows visible too. 

Not sure, this is okay too.